data/data/aws/vpc/vpc-private.tf

```diff
@@ -33,6 +48,9 @@ resource "aws_subnet" "private_subnet" {
 
   availability_zone = var.availability_zones[count.index]
 
+  ipv6_cidr_block                 = var.use_ipv6 == true ? cidrsubnet(data.aws_vpc.cluster_vpc.ipv6_cidr_block, 8, count.index) : ""
```

Comment:
https://github.com/openshift/installer/pull/2555/files#diff-2f4127d65f2b75935b1181f433f7bb75R47 looks different from that one?

Reply:
I can't tell what that link was to.  Which other change are you referring to?

Reply:
the IPV4 cidr is calculated as also similar question to https://github.com/openshift/installer/pull/2555#discussion_r354966038

my preference would be to keep it similar..

Reply:
I've added comments to hopefully help explain what this is doing.  Let me know if you have a more specific suggestion, since I'm not sure what you mean by wanting it similar to IPV4.